<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicTacToeClass</class>
 <widget class="QMainWindow" name="TicTacToeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TicTacToe</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Game"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionAccept_Move"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
   <property name="toolTip">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionAccept_Move">
   <property name="text">
    <string>Accept Move!</string>
   </property>
   <property name="toolTip">
    <string>Accept Move!</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TicTacToe.qrc"/>
 </resources>
 <connections/>
</ui>
